website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -349,10 +384,9 @@ To update the original datasource with the new data, you might execute a query l
 
 Assuming that the POST request is successful, the `myAPI` datasource receives the new information, and the Table should contain the new row the next time it updates.
 
-## What's next?
+## Further reference
 
-The following resources can come in handy as you need to learn new tricks:
+The following resources may come in handy:
 
-* [Core Concepts](./../../../core-concepts/connecting-to-data-sources/README.md)
-* [Appsmith Framework](../../appsmith-framework/)
-* [JavaScript Editor](../../../core-concepts/writing-code/javascript-editor-beta/)
+* [Sample apps for Tables](/learning-and-resources/sample-apps#table)
+  * [Editable table rows](https://app.appsmith.com/app/editable-table/row-save-62d8f8d0e1c2ed505a0557cc)
```

Comment:
As there is not a lot of stuff we are giving them to refer, we can remove this section and have the sample app (Editing table rows) linked below the properties and events table. @pranavkanade can we also add adding new rows to the sample app, please?

Reply:
If there are troubleshooting issues/errors related to inline editing, they should come here